is_core/templates/is_core/forms/responsive_inline_formset.html

```diff
@@ -1,15 +1,14 @@
 {% load i18n forms %}
 
 <fieldset class="inline responsive-inline {{ class_names|join:' ' }}">
-	{% block inline-title %}{% if title %}<legend>{{ title }}</legend>{% endif %}{% endblock %}
 	{{ formset.management_form }}
 	<div class="form">
 		{% if formset.all_forms %}
 			{% for form in formset.all_forms %}
 				{% if forloop.first %}
 					{% block inline-thead %}
 						<div class="inline-header form-inline">
-							{% for field_name in fieldset %}
+							{% for field_name in fields %}
```

Comment:
why it was named fieldset :D

Reply:
it is a set of fields? :clever:

Reply:
for me fieldset and fields are two different thinks. Because fieldset has tittle, can contain inline_view or another fieldset